deepcell_label/imgutils.py

```diff
@@ -1,26 +1,9 @@
 """Utilities for handling images"""
 import io
 
-import matplotlib.pyplot as plt
 import numpy as np
 from matplotlib.colors import Normalize
 from PIL import Image
-from skimage.segmentation import find_boundaries
-
-
-def pngify(imgarr, vmin, vmax, cmap=None):
-    out = io.BytesIO()
-
-    if cmap:
-        cmap = plt.get_cmap(cmap)
-        imgarr = Normalize(vmin=vmin, vmax=vmax)(imgarr)
-        # apply the colormap
-        imgarr = cmap(imgarr, bytes=True)
-
-    img = Image.fromarray(imgarr)
-    img.save(out, format='png')
-    out.seek(0)
-    return out
 
 
 def grayscale_pngify(imgarr):
```

Comment:
Can this one be computed on the frontend as well?

Reply:
Yes, I updated the backend to send the raw data in an array just like the labeled array and then computes the image on the client.